support/setup_codes.py

```diff
@@ -1207,14 +1236,54 @@ def subdirs_in_lib_dir(self):
                 yield path
                 
     
-    
-    
+    def copy_amuse_src(self):
+        """Copy the Amuse source code to the build directory
+
+        Only needed for Python 3, to be able to convert the Amuse main
+        code to Python 2.
+
+        """
+        srcdir = os.path.dirname(self.codes_src_dir)
+        destdir = os.path.dirname(self.codes_dir)
+        if srcdir != destdir:
+            owd = os.getcwd()
+            os.chdir(srcdir)
+            amusebuilddir = os.path.join(destdir, 'amuse')
+            # Only Python 3.5+ supports '**', with recursive=True
+            # (even if this function is likely only run by Python 3.5+)
+            files = glob.glob('*.py')
```

Comment:
Better to use a built-in distutils function that grabs these files from setup.py or MANIFEST.in.
Needs some digging into the distutils code, e.g. the build command.